lib/exq_scheduler/schedule/parser.ex

```diff
@@ -54,6 +55,20 @@ defmodule ExqScheduler.Schedule.Parser do
   end
 
   defp create_job(schedule) do
-    Map.drop(schedule, @non_job_keys) |> Poison.encode!()
+    job = Map.drop(schedule, @non_job_keys)
+    validate_config(job)
+    job = job |> Poison.encode!()
+  end
+
+  defp validate_config(job) do
+    if job["class"] == nil do
+      exit "class is not configured for scheduler"
```

Comment:
It's better to raise an exception with proper message. Add details like name, cron etc so it's easy to identify